vivisect/vector.py

```diff
@@ -22,7 +22,8 @@ def __init__(self):
         self.res = []
 
     def apicall(self, emu, op, pc, api, argv):
-        self.res.append( (op.va, argv) )
+        magicargs = [(val, emu.getMagic(val)) for val in argv]
```

Comment:
fuck, i thought i had tested this... i'm not sure why this didn't break things for me.  it was certainly breaking before.

Reply:
damn, that's the old magic subsystem, which i thought still had valid API's... but it has since become the Taint system.  i'm still baffled how this worked for my use-case.  i didn't change this because i had spare time.  it has been broken.

Reply:
oddly enough, this doesn't *break* outright (like my original attempts) ... but it obviously doesn't work.